### ISSUE TITLE
A Resume Training Issue

### ISSUE THREAD

**Issue by cuongnguyengit** [NVIDIA/tacotron2#485]
Hi,

I have a small dataset in my language for training with Tacotron2.
I trained 12k steps with batch size 64. Then i want to resume training more steps.
When i do it, loss always increate like the first training in step 0.  

 **Train loss 12001 0.166141** Grad Norm 0.372761 2.51s/it
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 32768.0
**Train loss 12002 4.972964** Grad Norm inf 1.78s/it
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 16384.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 8192.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 4096.0
Train loss 12005 5.318559 Grad Norm inf 1.75s/it
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 2048.0
Train loss 12006 4.378150 Grad Norm inf 1.76s/it
Train loss 12007 4.543454 Grad Norm 29.483887 1.85s/it
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 1024.0
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 512.0
Train loss 12010 8.917800 Grad Norm 196.323471 1.87s/it
Train loss 12011 5.221804 Grad Norm 41.242760 1.67s/it

This is my problem. Can you help me or explain for them?
In step 12k, my inference is quite good but not really natural.
After resume training, my inference is so bad in 26k step.

Thank you.

**Traing Loss**
![26k_loss_tacotron](https://user-images.githubusercontent.com/45245565/118375051-69162b00-b5e9-11eb-82bd-78aff5815ce4.PNG)

**Val Loss**
![26k_loss_tacotron_val](https://user-images.githubusercontent.com/45245565/118375052-6adfee80-b5e9-11eb-9515-bb8865660c46.PNG)

log for 26k steps
[log-tacotron2.zip](https://github.com/NVIDIA/tacotron2/files/6488049/log-tacotron2.zip)
